cartographer_rviz/cartographer_rviz/drawable_submap.cc

```diff
@@ -97,6 +108,13 @@ DrawableSubmap::DrawableSubmap(const ::cartographer::mapping::SubmapId& id,
       "" /* title */, visible, "" /* description */, submap_category,
       SLOT(ToggleVisibility()), this);
   submap_node_->attachObject(manual_object_);
+
+  // TODO: Make it toggleable.
+  submap_id_text_.setTextAlignment(::rviz::MovableText::H_CENTER,
+                                   ::rviz::MovableText::V_ABOVE);
+  submap_id_text_node_->setPosition(ToOgre(kSubmapIdHeight));
```

Comment:
I tried both `setLocalTranslation()` and `setGlobalTranslation()`. And they don't seem to show texts on expected position. Having child  node seems like the cleanest way of showing texts on axes above.